frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart

```diff
@@ -2,45 +2,95 @@ import 'dart:io';
 
 import 'package:appflowy/workspace/application/appearance.dart';
 import 'package:appflowy/workspace/application/home/home_setting_bloc.dart';
+import 'package:appflowy/workspace/application/tabs/tabs_bloc.dart';
 import 'package:flutter/material.dart';
 import 'package:hotkey_manager/hotkey_manager.dart';
 import 'package:provider/provider.dart';
 
+typedef KeyDownHandler = void Function(HotKey hotKey)?;
```

Comment:
Remove the ? after the typedef and declare it as optional or not, depending on the condition.